dc-model/src/main/java/de/digitalcollections/model/identifiable/entity/DigitalObject.java

```diff
@@ -49,62 +50,95 @@ public class DigitalObject extends Entity {
    * Sorted list of links (with description and MIME type) to human readable formats like a
    * permalink, OPAC/catalogue page, PDF download, ...
    */
-  private List<FileResource> renderingResources = new ArrayList<>(0);
+  private List<FileResource> renderingResources;
 
   /** version of the digital object. */
   private Version version;
 
   /** Default constructor, which also sets the EntityType to {@link EntityType#DIGITAL_OBJECT} */
   public DigitalObject() {
     super();
+    init();
+  }
+
+  @Override
+  protected void init() {
+    super.init();
     this.entityType = EntityType.DIGITAL_OBJECT;
+    if (fileResources == null) {
```

Comment:
Wieso "if"? Ist doch immer null....?

Reply:
wie besprochen, wenn die `init()`-Methode durch den Builder (in `prebuild`) aufgerufen wird, kann `fileResources` bereits nicht null sein, und darf daher nicht mit `null` überschrieben werden.